### ISSUE TITLE
New heading "Data type" in element descriptions 

### ISSUE THREAD

**Issue by karinbredenberg** [SAA-SDT/EAS-TagLibraries#62]
## Creator of issue

1. Karin Bredenberg, TS EAS co-chair

## The issue relates to

- [ ] EAC-CPF schema issue
- [ ] EAC-CPF Tag Library issue
- [ ] EAD schema issue
- [ ] EAD Tag Library issue
- [ ] Schema issue
- [x] Tag Library issue
- [ ] Suggestions for all schemas
- [ ] Suggestions for all Tag Libraries
- [ ] Other

## Wanted change/feature

* Text: Addition in the element descriptions of new heading "Data type" similar to the one for attributes. Gives updates needed in: TEI-schema, transformation x2, template.

**Comment by alexduryee** [comment]
Tested out locally and the XSLT/outputs pick this up fine already.

TODO: add to the TEI for each element, add to template

**Comment by kerstarno** [comment]
@alexduryee - could you confirm whether this update is currently mainly done for the TEI schema  and the transformations, but is pending inclusion in the template and addition to each element? Asking as I can see the "Data type" section for attributes, but not for elements in the draft PDF for the TL.

**Comment by alexduryee** [comment]
That's correct - the section works fine for elements, but currently no elements use it in EAD3 or the EAC revision.
